docker/scripts/sidecar.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
can we just use the existing `sbin` script?

Reply:
agree that this would be nice but the sbin script relies on a `configure.sh` script that isn't easily ported to the docker image without preserving the entire directory structure of the mono repo.
I think we should keep it for now and try to consolidate them when we have a bit more time